flight/Libraries/alarms.c

```diff
@@ -238,6 +238,150 @@ static int32_t hasSeverity(SystemAlarmsAlarmOptions severity)
     return 0;
 }
 
+static const char alarm_names[][8] = {
```

Comment:
Yah, I really think we should allocate it [9] and remove the magic to cope with the null not being there below.  Costs a few bytes of flash up here, but saves a few bytes of program code below.